cluster-operator/src/test/java/io/strimzi/operator/cluster/KafkaVersionTestUtils.java

```diff
@@ -14,25 +14,28 @@
 public class KafkaVersionTestUtils {
 
     public static final String KAFKA_VERSION_YAML_TEMPLATE =
-                    "- version: %s\n  format: %s\n  protocol: %s\n  checksum: %s\n  third-party-libs: %s\n  default: %b\n";
+                    "- version: %s\n  format: %s\n  protocol: %s\n  zookeeper: %s\n  checksum: %s\n  third-party-libs: %s\n  default: %b\n";
     public static final String KAFKA_IMAGE_STR = "strimzi/kafka:latest-kafka-";
     public static final String KAFKA_CONNECT_IMAGE_STR = "strimzi/kafka-connect:latest-kafka-";
     public static final String KAFKA_CONNECT_S2I_IMAGE_STR = "strimzi/kafka-connect-s2i:latest-kafka-";
     public static final String KAFKA_MIRROR_MAKER_IMAGE_STR = "strimzi/kafka-mirror-maker:latest-kafka-";
 
-    public static final String LATEST_KAFKA_VERSION = "2.3.1";
+    public static final String LATEST_KAFKA_VERSION = "2.3.0";
     public static final String LATEST_FORMAT_VERSION = "2.3";
     public static final String LATEST_PROTOCOL_VERSION = "2.3";
+    public static final String LATEST_ZOOKEEPER_VERSION = "3.5.4";
```

Comment:
How do these changes fit with this PR?

Reply:
This is part of the ZK upgrade PR. I originally had the changes to this class in there to help with the 2.4.0 testing and patched them into this PR as they are relevant to testing version changes. I thought I had removed all the ZK upgrade specific sections but I missed this one. I will fix this and double check that there aren't any more.